lib/Serialization/Deserialization.cpp

```diff
@@ -4507,7 +4507,11 @@ void ModuleFile::loadAllMembers(Decl *container, uint64_t contextData) {
   members.reserve(rawMemberIDs.size());
   for (DeclID rawID : rawMemberIDs) {
     Expected<Decl *> next = getDeclChecked(rawID);
-    if (!next) {
+    if (next) {
+      assert(next.get() && "unchecked error deserializing next member");
+      members.push_back(next.get());
+    }
+    else {
```

Comment:
Double nitpick:
- LLVM style puts `else` on the same line as the close brace.
- Early returns are encouraged, so if you're going to invert this `if` you may as well use `continue` to avoid the `else` altogether.